templates/SilverStripe/Blog/Model/Layout/Blog.ss

```diff
@@ -0,0 +1,31 @@
+<div class="container">
+    <div class="row">
```

Comment:
We have this snippet of code in every single layout ... why don't we move it to an include statement?

Reply:
Yeah, good call